src/main/java/games/loveletter/LoveLetterHeuristic.java

```diff
@@ -32,11 +32,12 @@ public double evaluateState(AbstractGameState gs, int playerId) {
         LoveLetterParameters llp = (LoveLetterParameters) gs.getGameParameters();
         Utils.GameResult playerResult = gs.getPlayerResults()[playerId];
 
-        if (playerResult == Utils.GameResult.LOSE)
-            return -1;
-        if (playerResult == Utils.GameResult.WIN)
-            return 1;
-
+        if (!gs.isNotTerminal()) {
```

Comment:
Is this necessary? If a player's result was set to lose/win, then they're no longer playing so don't need to do the more complex calculations

Reply:
Odd - the change log gives a strange view. If you look at the actual full code you'll see this is exactly the case.